crates/templates/src/manager.rs

```diff
@@ -629,6 +629,47 @@ mod tests {
             output_path: output_dir.clone(),
             name: "my project".to_owned(),
             values,
+            accept_defaults: false,
+        };
+
+        template
+            .run(options)
+            .silent()
+            .await
+            .execute()
+            .await
+            .unwrap();
+
+        let cargo = tokio::fs::read_to_string(output_dir.join("Cargo.toml"))
+            .await
+            .unwrap();
+        assert!(cargo.contains("name = \"my-project\""));
+    }
+
+    #[tokio::test]
+    async fn can_run_template_with_accept_defaults() {
+        let temp_dir = tempdir().unwrap();
+        let store = TemplateStore::new(temp_dir.path());
+        let manager = TemplateManager { store };
+        let source = TemplateSource::File(project_root());
+
+        manager
+            .install(&source, &InstallOptions::default(), &DiscardingReporter)
+            .await
+            .unwrap();
+
+        let template = manager.get("http-rust").unwrap().unwrap();
+
+        let dest_temp_dir = tempdir().unwrap();
+        let output_dir = dest_temp_dir.path().join("myproj");
+        let values = [("http-path".to_owned(), "/path/...".to_owned())]
```

Comment:
Suggestion: _don't_ send in a value for `http-path`, and assert that the `spin.toml` correctly gets the default value.